binr/r2r/r2r.c

```diff
@@ -209,7 +209,7 @@ int main(int argc, char **argv) {
 		for (i = 0; i < R_ARRAY_SIZE (streams); i++) {
 			GetConsoleMode (streams[i], &mode);
 			SetConsoleMode (streams[i],
-			                mode | ENABLE_PROCESSED_OUTPUT | ENABLE_VIRTUAL_TERMINAL_PROCESSING);
+				mode | ENABLE_PROCESSED_OUTPUT | ENABLE_VIRTUAL_TERMINAL_PROCESSING);
```

Comment:
no, this is fine, the indentaion is 2 tabs for continuation in the next line, see vim autoindent as reference, that's how it's been always, the problem is only if you mix spaces and tabs

Reply:
This line was indented with spaces. You can tell here since github indents tabs with 4 spaces, idk if that's a setting. I will change it to 2 tabs.